app/bundles/LeadBundle/EventListener/SegmentLogReportSubscriber.php

```diff
@@ -93,8 +93,12 @@ public function onReportGenerate(ReportGeneratorEvent $event)
                 $qb->expr()->isNotNull('log_removed.date_added')
             )
         );
-        $qb->setParameter('dateFrom', $event->getOptions()['dateFrom']->format('Y-m-d H:i:s'));
-        $qb->setParameter('dateTo', $event->getOptions()['dateTo']->format('Y-m-d H:i:s'));
+
+        $dateFrom = $event->getOptions()['dateFrom'] ?? (new \DateTime('-30 days'));
```

Comment:
If we use it in php.ini different timezone, then Datetime object return time in timezone from php.ini.
Need force set UTC time or use DateTimeHelper with toUtcString method.